apps/web/cypress/fixtures/questionable-test-data.json

```diff
@@ -0,0 +1,171 @@
+          "answer": "C"
+        },
+        {
+          "question": "P07",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P08",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P1",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P2",
+          "type": "radio",
+          "answer": "1B"
+        },
+        {
+          "question": "P3",
+          "type": "checkbox",
+          "answer": "1"
+        },
+        {
+          "question": "P6",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P9",
+          "type": "checkbox",
+          "answer": "1"
+        },
+        {
+          "question": "P11",
+          "type": "radio",
+          "answer": "3"
+        },
+        {
+          "question": "P12",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P13",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P17",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P18",
+          "type": "radio",
+          "answer": "true"
+        }
+      ],
+      "result": "R11"
+    },
+    {
+        "title": "doesn't successfully get to R10",
+        "steps": [
+          {
+            "question": "P01",
+            "type": "radio",
+            "answer": "yes"
+          },
+          {
+            "question": "P04",
+            "type": "radio",
+            "answer": "false"
+          },
+          {
+            "question": "P05",
+            "type": "radio",
+            "answer": "false"
+          },
+          {
+            "question": "P06",
+            "type": "checkbox",
+            "answer": "C"
+          },
+          {
+            "question": "P07",
+            "type": "radio",
+            "answer": "false"
+          },
+          {
+            "question": "P08",
+            "type": "radio",
+            "answer": "true"
+          },
+          {
+            "question": "P1",
+            "type": "radio",
+            "answer": "true"
+          },
+          {
+            "question": "P2",
+            "type": "radio",
+            "answer": "1"
+          },
+          {
+            "question": "P11",
+            "type": "radio",
+            "answer": "3"
+          },
+          {
+            "question": "P12",
+            "type": "radio",
+            "answer": "false"
+          },
+          {
+            "question": "P13",
+            "type": "radio",
+            "answer": "true"
+          },
+          {
+            "question": "P14",
+            "type": "radio",
+            "answer": "true"
+          },
+          {
+            "question": "P17",
+            "type": "radio",
+            "answer": "true"
```

Comment:
Same thing here, getting R20 instead of moving on to P18. Which is "correctly incorrect" haha

Reply:
@nicholaslyon-dt would love your or @wardm89's perspective on what we think the values  for these "questionable" flows here *should* be before we bring this to the ministry. I'm not sure if we are of the opinion that the feasibility report is wrong for these cases or we should interpret the logic differently. (Or if we think we are getting the correct answer from the tool and these flows aren't questionable at all.)

Reply:
Yeah I think we just need to bring up these flows with the ministry and ask. The feedback from the SME in an email was:

`ie. only check condition were both P(5) and P(8) are any value but true`

I forwarded y'all the mail with my set of questions and his answers if you want to take a look!

Reply:
One of the weird things about this flow is that both "yes" and "no" to P17 have the same result.